.github/labeler.yml

```diff
@@ -0,0 +1,14 @@
+core:
+  - all:
+    - '**'
+    - '!src/**'
+
+lang:en:
+  - all:
+    - 'src/**/*.md'
+    - '!src/ja/**/*.md'
+
+lang:ja:
+  - all:
+    - '!src/**/*.md'
+    - 'src/ja/**/*.md'
```

Comment:
I defined 3 labels: 

- `core` for PRs and issues related to application (or, anything than docs itself)
- `lang:en` for English contents
- `lang:ja` for Japanese contents

Note that I defined just as a starting point of discussion.
I'm glad to hear your thoughts.